### PR TITLE
Upgrade agent bond, migrate to yaml JMX exporter config

### PR DESCRIPTION
**Pull request by jimmidyson**

### CODE REVIEW

**Comment by rhuss** [comment]
LGTM

**Comment by jimmidyson** [comment]
Thanks @rhuss!

File: java/images/alpine/openjdk7/jdk/jmx_exporter_config.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+lowercaseOutputName: true
+lowercaseOutputLabelNames: true
+whilelistObjectNames:
```
Comment (rhuss) [inline java/images/alpine/openjdk7/jdk/jmx_exporter_config.yml:4]: oops, was too fast with my LGTM ;-)

Comment (jimmidyson) [inline java/images/alpine/openjdk7/jdk/jmx_exporter_config.yml:4]: Oh sh*t - sorry! Let me fix up...

**Pull request merged**